MapboxDirections/MBRouteLeg.swift

```diff
@@ -129,10 +134,13 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    open let segmentDistances: [CLLocationDistance]?
+    
+    open let segmentExpectedTravelTimes: [TimeInterval]?
     
-    open let segmentAttributes: [Attribute:[Any]]?
+    open let segmentSpeeds: [CLLocationSpeed]?
     
-    open let nodeAttributes: [Attribute:[Any]]?
+    open let nodeOpenStreetMapNodeIdentifiers: [Int]?
```

Comment:
To avoid overflow, OSM node IDs [must be stored as 64-bit integers](http://textual.ru/64/). [`Int`](https://developer.apple.com/reference/swift/int) can be either 32-bit or 64-bit according to the device’s architecture. To be on the safe side and to clarify our intent, use `Int64`.